internal/controller/utils.go

```diff
@@ -36,3 +38,16 @@ func MkdirTempAbs(dir, pattern string) (string, error) {
 	}
 	return tmpDir, nil
 }
+
+// HasChanged evaluates the given action and returns true
+// if the action type matches a resource mutation or deletion.
+func HasChanged(action ssa.Action) bool {
+	switch action {
+	case ssa.SkippedAction:
+		return false
+	case ssa.UnchangedAction:
```

Comment:
What about `ssa.UnknownAction`?

Reply:
UnknownAction would be accompanied by an error, which always should alert.